examples/BUILD

```diff
@@ -93,6 +93,41 @@ py_binary(
     ],
 )
 
+py_binary(
+  name = "rss_test",
+  srcs = ["rss_test.py"],
+  data = ['//bark:generate_core',
+          '//bark/runtime/tests:xodr_data',
+          ':params'],
+  imports = ['../python/'],
+  deps = ["//bark/runtime/commons:parameters",
+          "//bark/runtime/commons:xodr_parser",
+          "//bark/runtime/viewer:pygame_viewer",
+          "//bark/runtime/viewer:matplotlib_viewer",
+          "//bark/runtime:runtime",
+          "//bark/benchmark:benchmark_runner",
+          "@benchmark_database//load:benchmark_database",
+          "@benchmark_database//serialization:database_serializer",
+          "//bark/benchmark:benchmark_analyzer",
+          "//bark/runtime/scenario/scenario_generation:scenario_generation",
+          ],
+)
+
+py_binary(
```

Comment:
this binary should be optional and only work if RSS is included

Reply:
It is now implemented in these two commits https://github.com/bark-simulator/bark/commit/c28b2cd7a5ac44335772a324ba4c1efcbebc10ea https://github.com/bark-simulator/bark/commit/92023c78ed928236a5072cf48565776070683328